chaindata.json

```diff
@@ -2888,7 +2861,10 @@
     "id": "polkadex-polkadot",
     "name": "Polkadex",
     "account": "*25519",
-    "rpcs": ["wss://polkadex-parachain.api.onfinality.io/public-ws"],
+    "rpcs": [
```

Comment:
Polkadex parachain had only a single RPC which was an Onfinality node, so I replaced it with the two others listed on polkad.js.org/apps